packages/expo-dev-menu/ios/UITests/DevMenuTests.swift

```diff
@@ -69,7 +68,6 @@ class DevMenuTests: XCTestCase {
     assertViewExists(text: "Host:")
     assertViewExists(text: "localhost:1234")
     assertViewExists(text: "JS Engine:")
-    assertViewExists(text: "JavaScriptCore")
```

Comment:
bare expo now apparently uses Hermes, so this would need to be changed to `assertViewExists(text: "Hermes (bytecode v76)")` to pass (and would need to be updated each time the bytecode version changes). Better to leave the test agnostic of the environment.